### PR TITLE
feat: add gRPC and HTTPS ports to Consul array

### PR DESCRIPTION
**Pull request by bfqrst**
## Description
Default ports in Consul have changed over time.

## Motivation and Context
In order to correctly generate rules for a Consul security group, this should be considered.

## Breaking Changes
None that I know of.

## How Has This Been Tested?
- [x] I have tested and validated these changes referencing my local branch in my Consul deployments, which worked as expected.

### CODE REVIEW

**Comment by antonbabenko** [comment]
@bryantbiggs When I clicked "Update branch" on the PR the commit 779c0bb is pushed to the `master` branch of the fork which triggers a GH Action for release. Is there a way to trigger Release only on the main repo and not for forks?

**Comment by antonbabenko** [comment]
This PR is included in version 4.7.0 :tada:

**Comment by bryantbiggs** [comment]
@antonbabenko ya we could add in a hacky repo name check. however, unless users have the GH token set as a secret and as the same name, nothing will happen.

**Comment by antonbabenko** [comment]
Ok, in the future let's add such a check to not even trigger anything. I propose that we track such issues somewhere in [meta repository](https://github.com/terraform-aws-modules/meta) and apply them at once a bit later.

It is good that secrets are not floating around from the main repo to forks :)

File: rules.tf [hunk 1]
```diff
@@ -26,15 +26,16 @@ variable "rules" {
...
+    consul-grpc-tcp        = [8502, 8502, "tcp", "Consul gRPC"]
+    consul-webui-http-tcp  = [8500, 8500, "tcp", "Consul web UI HTTP"]
+    consul-webui-https-tcp = [8501, 8501, "tcp", "Consul web UI HTTPS"]
+    consul-dns-tcp         = [8600, 8600, "tcp", "Consul DNS"]
+    consul-dns-udp         = [8600, 8600, "udp", "Consul DNS"]
+    consul-serf-lan-tcp    = [8301, 8301, "tcp", "Serf LAN"]
+    consul-serf-lan-udp    = [8301, 8301, "udp", "Serf LAN"]
+    consul-serf-wan-tcp    = [8302, 8302, "tcp", "Serf WAN"]
+    consul-serf-wan-udp    = [8302, 8302, "udp", "Serf WAN"]
     # Docker Swarm
```
Comment (analytically) [inline rules.tf:39]: 8300 is still used?

**Pull request merged**